client/components/CustomBankField.js

```diff
@@ -0,0 +1,66 @@
+import {has, translate as t} from '../Helpers';
+import T from './Translated';
+
+export default class CustomBankField extends React.Component {
+
+    constructor(props) {
+        has(props, 'params');
+
+        super(props);
+    }
+
+    getValue() {
+        let node = this.refs[this.props.params.name].getDOMNode();
+        return {
+            name: this.props.params.name,
+            value: (this.props.params.type === "number") ? parseInt(node.value, 10) : node.value
+        };
+    }
+
+    setValue(value) {
```

Comment:
setValue is only used to reset the input value as in https://github.com/bnjbvr/kresus/blob/incoming/client/components/Settings.js#L60.
However, you're right, the current code won't work.

You are correct. I assumed if we don't show the password, maybe the others fields value should not be displayed either ?